charts/assemble-backstage/templates/postgres-pvc.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.postgres.persistent }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name:  {{ include "assemble-backstage.fullname" . }}-postgres
+  labels:
+    paas.redhat.com/appcode: "ASBL-001"
+  annotations:
+    kubernetes.io/reclaimPolicy: Delete 
+spec:
+  storageClassName: aws-efs-tier-c3
```

Comment:
In my testing, I needed to be able to configure this to make things work, so I think we should go ahead and make it a configurable value.

Reply:
Made configurable